.github/workflows/publish-on-tag.yml

```
@@ -0,0 +1,72 @@
+      - name: Publish to GitHub Packages
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
+        with:
+          name: Release v${{ env.VERSION }}
+          body: |
+            ## Changes in this Release
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)